src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -105,13 +136,13 @@ public void start(Stage primaryStage) throws Exception {
         // menuBar.getMenus().addAll(menuConvo);
 
         ObservableList<String> usersList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+                "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
```

Comment:
Can we get rid of these hardcoded names yet? Are we ready to connect to the database?